jans-cedarling/cedarling/src/jwt/test/with_validation/access_token.rs

```diff
@@ -105,9 +106,13 @@ fn test_missing_claim(missing_claim: &'static str) {
     });
 
     // generate the signed token strings
-    let access_token = generate_token_using_claims(&access_token_claims, &encoding_keys[0]);
-    let id_token = generate_token_using_claims(&id_token_claims, &encoding_keys[1]);
-    let userinfo_token = generate_token_using_claims(&userinfo_token_claims, &encoding_keys[0]);
+    let (access_token, id_token, userinfo_token) =
```

Comment:
please use structure but not tuple

Reply:
refactored here [aea5e9f](https://github.com/JanssenProject/jans/pull/9982/commits/aea5e9f819a9187287d9d13d7ee8f68c98198b06)